react-training/src/app/constants/auth.js

```diff
@@ -0,0 +1,3 @@
+export const AUTHENTICATED_DEFAULT_ROUTE = '/';
+
+export const UNAUTHENTICATED_DEFAULT_ROUTE = '/login';

```

Comment:
Move `constants` folder one level higher to `src`. All constants must be generics for all app, like the components, redux, services, etc